Types.h

```diff
@@ -5,6 +5,35 @@
 
 #include <MY17_Can_Library.h>
 
+// All names in the enum are taken from the NDrive manual
+typedef enum {
```

Comment:
I created the enum because I wanted a type that represents sending no motor controller requests ("NO_MOTOR_CONTROLLER_REQUEST" in this enum). This type is used once the VCU has finished sending permanent transmit requests